docs/project-workflow.md

```diff
@@ -0,0 +1,284 @@
+# Project Workflow
+
+## Dependencies
+
+* `~/.gen3/credentials.json` [identity file](https://gen3.org/resources/user/using-api/#credentials-to-send-api-requests) from the [portal](https://aced-training.compbio.ohsu.edu/identity)
+* [gen3-client](https://gen3.org/resources/user/gen3-client/#1-installation-instructions)
+
+## Usage:
+
+`gen3_util projects [OPTIONS] COMMAND [ARGS]...`
+
+  The following `--parameters` and environmental variables are synonymous:
+
+    * OBSERVATION_ID
+    * PATIENT_ID
+    * PROJECT_ID
+    * SPECIMEN_ID
+    * TASK_ID
+
+  You may set them as environmental variables or pass them as parameters to the command line.
+
+    * `--observation_id=observation-1` or `export OBSERVATION_ID=observation-1`
+
+  The following parameters may be used to control system wide behavior:
+
+    * GEN3_UTIL_CONFIG: Path to config file. [default: None]
+    * GEN3_UTIL_FORMAT: Output format. [default yaml]
+    * GEN3_UTIL_STATE_DIR: Path for logs and state information.  [default: ~/.gen3/gen3_util]
+    * GEN3_API_KEY: location of credentials.json file. [default: ~/.gen3/credentials.json]
+
+
+## Create a project in authorization system
+
+```text
+export PROJECT_ID=test-myproject
+# request a project create a project
+gen3_util projects new
+
+# user with appropriate authority signs requests for the project
+# optionally, use the `--username nancy@example.com` to limit the approvals to a specific user
+gen3_util access sign
+
+```
+
+# Use case: upload files and associate them with a study
+
+
+## add files to the manifest
+
+```text
+# find all files under tests/fixtures/add_files_to_study
+find tests/fixtures/add_files_to_study -type f  | xargs  -I PATH gen3_util files manifest put PATH
+
+```
+
+```text
+# list the file names in the upload manifest
+$ gen3_util files manifest ls | grep file_name
+  file_name: tests/fixtures/dir_to_study/file-1.txt
+  file_name: tests/fixtures/add_files_to_study/sub-dir/file-4.tsv
+  file_name: tests/fixtures/add_files_to_study/sub-dir/file-3.pdf
+  file_name: tests/fixtures/add_files_to_study/sub-dir/file-5
+  file_name: tests/fixtures/add_files_to_study/file-1.txt
+  file_name: tests/fixtures/add_files_to_study/README.md
+  file_name: tests/fixtures/add_files_to_study/file-2.csv
+
+```
+
+## upload and index the files
+
+```text
+gen3_util files manifest upload
+
+```
+
+## Create project metadata
+
+```text
+# create basic, minimal metadata for the project
+gen3_util meta create indexd /tmp/$PROJECT_ID
```

Comment:
If the command is only creating metadata and not directly interacting with the gen3 instance in any way, especially not with indexd then it is a bit misleading having indexd in the name. Some three word combination like `gen3_util meta create` or simpler would be better since it is creating metadata locally only.

Reply:
done